relay-server/src/actors/processor/span/processing.rs

```diff
@@ -128,18 +130,30 @@ pub fn extract_from_event(state: &mut ProcessEnvelopeState) {
             Ok(span) => span,
             Err(e) => {
                 relay_log::error!("Invalid span: {e}");
+                state.managed_envelope.track_outcome(
+                    Outcome::Invalid(DiscardReason::InvalidSpan),
+                    relay_quotas::DataCategory::SpanIndexed,
+                    1,
+                );
```

Comment:
This raises a new question: Do we want outcomes for spans that were extracted from transactions?

This question is especially relevant for the `Accepted` outcome. When it becomes time to charge customers for standalone spans, won't we need to differentiate between spans that were submitted as part of transactions vs. spans that were submitted standalone? cc @indragiek

Reply:
Alright!